tco.py

```
@@ -8,23 +8,68 @@
+        if debug:
+            print("ERROR: File is empty: {}".format(filename))
+        return False
+    if debug:
+        print("DEBUG: File exists and is not empty: {}".format(filename))
 
     return True
 
 
+# Read file as buffer
+def getfile(filename: str, debug: bool):
+    filehandle = open(filename, "r")
```

Ruff

Use a context manager for opening files

(SIM115)